definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -23,6 +23,7 @@
     "PreConditionState(s)": "caseAdded",
     "PostConditionState": "applicationSubmitted",
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/case-orchestration/assign-applicant-solicitor",
+    "RetriesTimeoutAboutToStartEvent": "3,4,5",
```

Comment:
Don't need to add it - default behaviour is to retry 3 times, but it won't break anything